app/controllers/api/v1/agencies_controller.rb

```diff
@@ -27,10 +27,10 @@ def index
           location = params[:coordinates] if params[:coordinates].present?
         end
 
-        distance = params[:distance] if params[:distance].present?
-        distance = 20 unless distance.present?
+        distance = params[:distance].present? ? params[:distance] : 20
```

Comment:
move the distance search outside of search

Reply:
> move the distance search outside of search

@micronix just wanted to double check before I move the distance out of the `params[:search]` loop. It might fail because we need `distance` for geocoder `near` method!